ddp-workspace/projects/ddp-dsm-ui/src/app/STORE/selectors/Utils/participantsList.util.ts

```diff
@@ -11,9 +11,51 @@ export const generateParticipantsList = (participants: ParticipantModel[], setti
     birthdate: '01/01/2022',
     registered: '02/02/2002',
     lastUpdated: '01/01/2022',
-    activities: pt.esData.activities.map(activity => ({
-      activityName: Object.values(actDefs).find(actDef => actDef['activityCode'] === activity.activityCode)['activityName'],
-      activityGuid: activity.guid
-    }))
+    activities: pt.esData.activities.map(activity => {
+      const activityGot = Object.values(actDefs).find(actDef => actDef['activityCode'] === activity.activityCode);
+      return {
+        name: activityGot['activityName'],
```

Comment:
**_#minor_**

just wondering why you use the format `activityGot['activityName']` (instead of `activityGot.activityName`).
At the same time `activity.guid`.

That is not an issue just interesting.